samples/go/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Isn't this in the image?

Reply:
This is the public certificate, which depending on the sample, gets copied to the image's system folder for certs (in go's case) or gets used directly in the same (in js's case) or etc.